README.md

```diff
@@ -1126,6 +1126,16 @@ mysetting: |
 
 The possibility is endless. Try importing values from your golang app, bash script, jsonnet, or anything!
 
+Then `execEnvs` same as `exec`, but it can receive a dict as the envs.  
+
+A usual usage of `execEnvs` would look like this:  
+
+```yaml
+mysetting: |
+{{ execEnvs (dict "envkey" "envValue") "./mycmd" (list "arg1" "arg2" "--flag1") | indent 2 }}
```

Comment:
@yxxhero @itscaro I think it's a bit hard to remember the arg ordering! How about naming it `envExec` after the fact that it receives the dict of envvars first, and then command and args in the same manner as `exec`?

Reply:
ok @mumoshu I change it right row.